docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -4169,4 +4169,47 @@ Again, when executed without any arguments, a help screen is shown.
 sys::[/usr/local/pf/bin/pfcmd_vlan -help]
 ----
 
+[appendix]
+Restoring a Percona XtraBackup dump
+-----------------------------------
+
+If you enabled Percona XtraBackup for your nightly backup, you can use the following instructions to restore it. In this example we will be restoring `/root/backup/packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream.gz`
+
+First, create a restore directory, move the backup to it and gunzip the backup:
+
+  # mkdir /root/backup/restore
+  # cd /root/backup/restore
+  # cp ../packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream.gz .
+  # gunzip packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream.gz
+
+Then extract the xbstream data:
+
+  # xbstream -x < packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream
+
+Once done, you should have a lot of files that were extracted in the restore dir. Now, lets place the xbstream back in the backup directory
+
+  # mv packetfence-db-dump-innobackup-2016-12-20_00h31.xbstream ../
+
+Next, install qpress (available from the percona repository) and process any qp file that were extracted:
+
+  # yum install qpress --enablerepo=percona-release-`uname -m`
```

Comment:
Something like that:
apt-key adv --keyserver keys.gnupg.net --recv-keys 1C4CBDCDCD2EFD2A
echo 'deb http://repo.percona.com/apt VERSION main' >> /etc/apt/sources.list
echo 'deb-src http://repo.percona.com/apt VERSION main' >> /etc/apt/sources.list
apt-get update
apt-get install qpress